### ISSUE TITLE
SheetBar is overflowing.

### ISSUE THREAD

**Issue by RackweLLizm** [antvis/G6VP#106]
Hello, we continue to use GraphInsight actively in our projects. When we activate the SheetBar feature, the left navigation bar we use in Layout overflows. We would be glad if you could provide information on this subject.
![overflow](https://user-images.githubusercontent.com/52632318/213464576-6c961a40-1cce-477d-b5ec-77390915d47d.png)

**Comment by pomelo-nwu** [comment]
@RackweLLizm Did you visit this website: https://graphinsight.antgroup.com ?
![image](https://user-images.githubusercontent.com/10703060/213582817-485540af-8baf-4ac9-bf9e-aaab18aced59.png)

I didn't reproduce the problem. I suggest you click "Download project", I will restore the project file, so as to reproduce the problem

**Comment by RackweLLizm** [comment]
@pomelo-nwu  

Yes, I visited the site and I'm designing my entire project on https://graphinsight.antgroup.com, but I still encountered this problem.

I suspect I have infrastructure built with Create React App (CRA) and it doesn't run LESS files directly. I have to run npm inject to do this, but for my project integrity I see it as risky. For this reason, I converted the .less files of Gi-Assets-Basic and Gi-Assets-Advanced libraries to .css and copied them onto index.css.

Is there a .less file for the Sheetbar that I've overlooked at this point? I think I checked them all one by one and added them completely.
